manifests/m/MimecastLtd/MimecastforOutlook/7.10.1.133/MimecastLtd.MimecastforOutlook.locale.en-US.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: MimecastLtd.MimecastforOutlook
+PackageVersion: 7.10.1.133
+PackageLocale: en-US
+Publisher: Mimecast Ltd
+PublisherUrl: https://www.mimecast.com/
+PublisherSupportUrl: https://community.mimecast.com/s/contactsupport
+PackageName: Mimecast for Outlook
+PackageUrl: https://community.mimecast.com/s/mimecast-for-outlook
+License: Copyright (c) Mimecast Ltd. All rights reserved

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate